sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/controllers/SpringAiOrchestrationController.java

```diff
@@ -0,0 +1,74 @@
+package com.sap.ai.sdk.app.controllers;
+
+import static com.sap.ai.sdk.orchestration.OrchestrationAiModel.GPT_35_TURBO;
+
+import com.sap.ai.sdk.orchestration.DpiMasking;
+import com.sap.ai.sdk.orchestration.OrchestrationModuleConfig;
+import com.sap.ai.sdk.orchestration.model.DPIEntities;
+import com.sap.ai.sdk.orchestration.spring.OrchestrationChatModel;
+import com.sap.ai.sdk.orchestration.spring.OrchestrationChatOptions;
+import java.util.Map;
+import lombok.val;
+import org.springframework.ai.chat.client.ChatClient;
+import org.springframework.ai.chat.client.advisor.MessageChatMemoryAdvisor;
+import org.springframework.ai.chat.memory.InMemoryChatMemory;
+import org.springframework.ai.chat.model.ChatModel;
+import org.springframework.ai.chat.model.ChatResponse;
+import org.springframework.ai.chat.prompt.Prompt;
+import org.springframework.ai.chat.prompt.PromptTemplate;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+/** Endpoints for the Orchestration service */
+@RestController
+@RequestMapping("/orchestration")
+class SpringAiOrchestrationController {
```

Comment:
(Comment)

This controller class is inconsistent with other controller classes. I'm not yet sure what's the best practice / expected class setup.

Reply:
I tried adding a service but it did not fix the `Type definition error: UserMessage`

Reply:
fixed